src/main/java/de/mediathekview/mserver/crawler/orf/tasks/OrfFilmDetailTask.java

```diff
@@ -0,0 +1,227 @@
+import de.mediathekview.mserver.crawler.orf.parser.OrfVideoDetailDeserializer;
+import java.lang.reflect.Type;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import mServer.crawler.CrawlerTool;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+
+public class OrfFilmDetailTask extends AbstractDocumentTask<Film, OrfTopicUrlDTO> {
+  
+  private static final Logger LOG = LogManager.getLogger(OrfFilmDetailTask.class);
+  
+  private static final String TITLE_SELECTOR = "h3.video_headline";
+  private static final String BROADCAST_SELECTOR = "div.broadcast_information";
+  private static final String TIME_SELECTOR = BROADCAST_SELECTOR + " > time";
+  private static final String DURATION_SELECTOR = BROADCAST_SELECTOR + " > span.meta_duration";
+  private static final String DESCRIPTION_SELECTOR = "div.details_description";
+  private static final String VIDEO_SELECTOR = "div.jsb_VideoPlaylist";
+  private static final String EPISODE_SELECTOR = "li.jsb_PlaylistItemFullscreen";
+  
+  private static final String ATTRIBUTE_DATETIME = "datetime";
+  private static final String ATTRIBUTE_DATA_JSB = "data-jsb";
+
+  private static final DateTimeFormatter DATE_TIME_FORMATTER =
+    DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+  
+  private static final Type OPTIONAL_VIDEOINFO_TYPE_TOKEN = new TypeToken<Optional<OrfVideoInfoDTO>>() {}.getType();
+  private static final Type OPTIONAL_EPISODEINFO_TYPE_TOKEN = new TypeToken<Optional<OrfEpisodeInfoDTO>>() {}.getType();
+  
+  public OrfFilmDetailTask(final AbstractCrawler aCrawler,
+      final ConcurrentLinkedQueue<OrfTopicUrlDTO> aUrlToCrawlDTOs) {
+    super(aCrawler, aUrlToCrawlDTOs);
+  }
+  
+  @Override
+  protected void processDocument(OrfTopicUrlDTO aUrlDTO, Document aDocument) {
+    final Optional<String> title = HtmlDocumentUtils.getElementString(TITLE_SELECTOR, aDocument);
+    final Optional<LocalDateTime> time = parseDate(aDocument);
+    final Optional<Duration> duration = parseDuration(aDocument);
+    final Optional<String> description = HtmlDocumentUtils.getElementString(DESCRIPTION_SELECTOR, aDocument);
+    
+    final Optional<OrfVideoInfoDTO> videoInfoOptional = parseUrls(aDocument);
+    
+    createFilm(aUrlDTO, videoInfoOptional, title, description, time, duration);
+    
+    final List<OrfEpisodeInfoDTO> episodes = parseEpisodes(aDocument);
+    episodes.forEach(episode -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)